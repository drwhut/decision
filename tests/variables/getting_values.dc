Start~#1

[Variable(myIntegerVar, Integer, 69)]
[Variable(myFloatVar, Float, 420.69)]
[Variable(myStringVar, String, "Hello, world!")]
[Variable(myBooleanVar, Boolean, true)]

< getting integer variable value >
myIntegerVar~#2
Print(#1, #2)~#3

< getting float variable value >
myFloatVar~#4
Print(#3, #4)~#5

< getting string variable value >
myStringVar~#6
Print(#5, #6)~#7

< getting boolean variable value >
myBooleanVar~#8
Print(#7, #8)~#9