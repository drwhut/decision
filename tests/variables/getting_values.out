69
420.69
Hello, world!
true
